### ISSUE TITLE
Combobox styling for single selection

### ISSUE THREAD

**Issue by feitzi** [vmware-clarity/ng-clarity#261]
## Describe the bug
If you use combox with a defined styling, the styling for the selected value only works if you enable `multiselect`. If you chose single selection the styling of the selected value will be ignored.

## How to reproduce
~~~ html
 <clr-combobox [clrMulti]="false" >
      <ng-container *clrOptionSelected="let selected">
        <clr-icon shape="house"></clr-icon>
        {{ selected }}
      </ng-container>
      <clr-options>
        <clr-option clrValue="available">
          <clr-icon shape="house"></clr-icon>
          Available
        </clr-option>
        <clr-option clrValue="pending">
          <clr-icon shape="house"></clr-icon>
          Pending
        </clr-option>
      </clr-options>
    </clr-combobox>
~~~

This code displays the following:
![image](https://user-images.githubusercontent.com/2361489/181707443-dc2c4d2d-860d-4e06-bb3b-62865629f23e.png)

## Expected behavior
If you select a value, the value should be displayed as given in the optionSelected container. Currently, the original value is displayed without templating.

## Versions

**Clarity version:**

- [ ] v12.x
- [x] v13.x
- [ ] Other:

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

## Notes
Is there maybe another component in clarity that supports styling for a single select box? The Select component of clarity doesn't support styling in general?

**Comment by Jinnie** [comment]
The single selection combobox is following the W3C A11Y pattern here:
https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/combobox/aria1.1pattern/listbox-combo.html
It does not allow to have a complex selection representation, as the selection is displayed in the search input box.

We won't be changing it without request from our Accessibility team.
cc: @amylinow

**Comment by Jinnie** [comment]
As a workaround you could use the multiselect combobox and programmatically make sure that all old values are removed when selecting a new one.
